sdk/communication/Azure.Communication.CallAutomation/tests/CallMedias/CallMediaAutomatedLiveTests.cs

```diff
@@ -53,7 +54,7 @@ public async Task ContinuousRecognitionTest()
 
             // answer the call
             var answerCallOptions = new AnswerCallOptions(incomingCallContext, new Uri(TestEnvironment.DispatcherCallback));
-            var answerResponse = await targetClient.AnswerCallAsync(answerCallOptions);
+            var answerResponse = await client.AnswerCallAsync(answerCallOptions);
```

Comment:
Is it normal that "client" is doing CreateCall and "client" is doing AnswerCall? If this is correct, then the code to create "targetClient" above should be removed.

Reply:
done, thanks!